### DISCUSSION TITLE
How to add inforation to wiki?

### DISCUSSION THREAD

**Discussion by INOPIAE** [rubberduck-vba/Rubberduck#6074] [General]
While setting up a new development system with VS 2022 I saw some information missing or out of date on [https://github.com/rubberduck-vba/Rubberduck/wiki/Contributing](https://github.com/rubberduck-vba/Rubberduck/wiki/Contributing).
How can I add the missing information?

**Comment by Vogel612** [comment]
A while back we unfortunately had to restrict editing access to the wiki, because someone had submitted multiple spam edits.

I've invited you to the repository with write privileges, I hope that is sufficient to gain access to the wiki

**Reply by INOPIAE** [reply]
It works
